tests/test_models.py

```diff
@@ -1358,3 +1361,170 @@ def test_current_framework_agreement_with_countersigned_and_new_signed(self):
             )
             supplier_framework = self.get_supplier_framework()
             assert supplier_framework.current_framework_agreement.id == 5
+
+
+class TestDirectAwardProjects(BaseApplicationTest, FixtureMixin):
+    _direct_award_project_name = ''
+
+    def test_create_new_project(self):
+        with self.app.app_context():
+            self.setup_dummy_user(role='buyer')
+
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+
+            assert project.id is not None
+            assert project.name == DIRECT_AWARD_PROJECT_NAME
+            assert isinstance(project.created_at, datetime)
+            assert project.completed_at is None
+            assert project.active is True
+
+    def test_serialize_project_contains_required_keys(self):
+        with self.app.app_context():
+            self.setup_dummy_user(role='buyer')
+
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+
+            project_keys_set = set(project.serialize().keys())
+            assert {'id', 'name', 'created_at', 'completed_at', 'status', 'active'} <= project_keys_set
+
+    def test_create_new_project_creates_project_users(self):
+        with self.app.app_context():
+            self.setup_dummy_user(role='buyer')
+
+            assert len(DirectAwardProjectUser.query.all()) == 0
+
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+
+            assert len(DirectAwardProjectUser.query.all()) == len(User.query.all())
+
+    def test_direct_award_project_statuses(self):
+        with self.app.app_context():
+            project = DirectAwardProject(name=DIRECT_AWARD_PROJECT_NAME, users=User.query.all())
+            db.session.add(project)
+            db.session.commit()
+            assert project.status == 'live'
+
+            project.active = False
+            db.session.add(project)
+            db.session.commit()
+            assert project.status == 'abandoned'
+
+            project.completed_at = datetime.utcnow()
+            db.session.add(project)
+            db.session.commit()
+            assert project.status == 'completed'
```

Comment:
So I'm thinking in terms of "creating a list" being the "completed" step.

But asking for a big refactoring now to have "list" as a first-class concept seems like a big ask at this point in the sprint.

So maybe the middle-ground approach is to split the "status" field into two, I'd suggest:

* active (boolean, not null)
* list_created_at (datetime, nullable)

Explanation: we don't need a separate time field, as we can derived whether or not the list was created from whether the datetime was set.